README.md

```diff
@@ -2,7 +2,9 @@
 Welcome to the codebase for the Metabolic Atlas project.
 
 The front-end uses [Vue.js](https://vuejs.org), with help of [Vue CLI](https://cli.vuejs.org/). The backend uses [Django REST framework](http://www.django-rest-framework.org) with [PostgreSQL](https://www.postgresql.org) as the database.  
-To learn more about the project, including database setup, support for multiple databases, and testing, please visit the [wiki](https://github.com/SysBioChalmers/MetabolicAtlas/wiki).
```

Comment:
since the wiki is not private and the link removed, I think a section about how to import the models should be added, next update, to this readme

Reply:
added to MA-834